src/main/msp/msp_box.c

```diff
@@ -75,7 +75,7 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { .boxId = BOXBLACKBOX, .boxName = "BLACKBOX", .permanentId = 26 },
     { .boxId = BOXFAILSAFE, .boxName = "FAILSAFE", .permanentId = 27 },
     { .boxId = BOXAIRMODE, .boxName = "AIR MODE", .permanentId = 28 },
-    { .boxId = BOX3D, .boxName = "3D DISABLE / SWITCH", .permanentId = 29},
+    { .boxId = BOX3D, .boxName = "3D DISABLE", .permanentId = 29},
```

Comment:
```suggestion
    { .boxId = BOX3DDISABLE, .boxName = "3D DISABLE", .permanentId = 29},
```
unless i'm wrong.

Reply:
`boxId` is correct. Just checked with new `boxName`

![image](https://github.com/user-attachments/assets/46befecc-15f3-41a3-b9d4-d19db26f8604)